flocker/node/_docker.py

```diff
@@ -242,6 +242,32 @@ def list():
         :return: ``Deferred`` firing with ``set`` of :class:`Unit`.
         """
 
+    def _image_data(image, container, state):
```

Comment:
The reason they ended up here is because `NamespacedDockerClient` proxies this interface through its `_client` attribute - have a look at the difference between this and my first commit to the branch and you can get an idea of how I was originally approaching the problem, but that doesn't work for the namespaced client tests. Any suggestions on how best to proceed?